cmd/bnsd/app/app.go

```diff
@@ -118,19 +122,43 @@ func Stack(issuer weave.Address, minFee coin.Coin) weave.Handler {
 	return Chain(authFn, minFee).WithHandler(Router(authFn, issuer))
 }
 
+// CronStack wires up a standard router with a cron specific decorator chain.
+// This can be passed into BaseApp.
+// Cron stack configuration is a subset of the main stack. It is using the same
+// components but not all functionalities are needed or expected (ie no message
+// fee).
+func CronStack() weave.Handler {
+	authFn := cron.Authenticator{}
```

Comment:
It took me a while to figure this out.

We cannot use `sigs` or `multisig` authentication decorator. This is because the application is not able to sign the message. Instead, I use `cron.Authenticator` to pass around the authentication condition. There is no signature checking involved.

Reply:
Yeah, I saw that below, that the what I thought would need `cron.NewAuthDecorator()` was done also in `x/cron.tick` I do agree that is cleaner way to do it.